site/docs/messaging/campaign-management/csp/brand-vetting-api.mdx

```diff
@@ -0,0 +1,368 @@
+</BrandVettingResponse>Z
+```
+
+### Error Response
+
+```http
+HTTP/1.1 400 Bad Request
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/vetting
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<CampaignSettingsResponse>
+    <ResponseStatus>
+        <ErrorCode>1003</ErrorCode>
+        <Description>Phone is required</Description>
+    </ResponseStatus>
+</CampaignSettingsResponse>
+```
+
+### Error Codes
+```http
+HTTP/1.1 400 Bad Request
+HTTP/1.1 403 Unauthorized
+HTTP/1.1 429 Too Many Requests
+```
+
+## Fetch External Vetting Records
+
+#### Request URL
+<Highlight color="#079CEE">GET</Highlight> <code>https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/vetting</code>
+
+### GET external vetting
+
+```http
+GET https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/vetting HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/vetting
+
+<?xml version="1.0" encoding="UTF-8"?>
+<BrandVettingsResponse>
+   <BrandVettings>
+      <BrandVetting>
+         <EvpId>WMC</EvpId>
+         <VettingId>531D1FD09D68401EA0E4624B8B7D4A14</VettingId>
+         <VettingScore>88</VettingScore>
+         <VettingClass>STANDARD</VettingClass>
+         <VettingStatus>ACTIVE</VettingStatus>
+         <CreateDate>2021-11-22T19:58:08</CreateDate>
+         <VettedDate>2021-11-22T19:58:38</VettedDate>
+      </BrandVetting>
+      <BrandVetting>
+         <EvpId>AEGIS</EvpId>
+         <VettingId>2ee15ee2-8e77-4c5c-ed14-08d99db075c6</VettingId>
+         <VettingScore>80</VettingScore>
+         <VettingClass>STANDARD</VettingClass>
+         <VettingStatus>ACTIVE</VettingStatus>
+         <CreateDate>2021-11-22T18:13:37</CreateDate>
+         <VettedDate>2021-11-22T18:13:37</VettedDate>
+      </BrandVetting>
+   </BrandVettings>
+</BrandVettingsResponse>
+```
+
+### Error Response
+
+```http
+HTTP/1.1 404 Not Found
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/brands/{brandId}/vetting
+
+<?xml version="1.0" encoding="UTF-8"?>
+<BrandVettingResponse>
+   <ResponseStatus>
+      <ErrorCode>12185</ErrorCode>
+      <Description>A call to the campaign registry service has failed validation.</Description>
+   </ResponseStatus>
+   <Errors>
+      <Error>
+         <Code>525</Code>
+         <Description>Vetting re-submission not allowed with same brand information.</Description>
+      </Error>
+   </Errors>
+</BrandVettingResponse>
+```
+
+### Error Codes
+```http
+HTTP/1.1 403 Unauthorized
+HTTP/1.1 404 Not Found
+HTTP/1.1 429 Too Many Requests
+```
```

Comment:
This isnt super consistent with how weve done other API guides on the site - not the end of the world but we might want to clean it up 

Generally the markdown format is as follows 

```
## Request External Vetting 

A high level overview of what the section is going to illustrate 

### Request URL

<Highlight color="#079CEE">POST</Highlight> <code>https://dashboard.bandwidth.com/api/...</code>

### Request Parameters 

(if applicable) a markdown table of the request body params in the following format 

| Parameter | Required | Description | 
|:----------|:----------:|:------------|
| `paramName` | Yes | What the parameter controls | 

### Examples 

If there is a single example you can follow this format 

> Request 

`xml 
<TheRequest>
`

> Response 

`xml 
<TheResponse/>
`

and if there are multiple examples feel free to use a tabs component to show an example of successful and failed responses if there is a need to illustrate the difference 

Then repeat for the next API operation 

## Import External Vetting Records 

### Request URL 
```